.devcontainer.json

```diff
@@ -37,4 +37,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
A few minor things like this were included because the linter are new, if preferred I can split those out from this PR.